app/assets/javascripts/short-form/components/file-uploader.html.slim

```diff
@@ -0,0 +1,39 @@
+.form-group.text-center ng-class="{ error: $ctrl.inputInvalid($ctrl.fileInputName) }" ng-hide="$ctrl.hasFile()"
```

Comment:
any reason why you haven't updated the preference file uploaders to use this component? Just too much refactoring for 1 PR?

Reply:
Preferences are more complicated, they have selects, after actions, etc. File uploader is a simplified version of preference file uploader, so it felt like it would require too many ifs.